crates/matrix-sdk/src/client/mod.rs

```diff
@@ -952,6 +952,48 @@ impl Client {
         self.login_with_token(token.as_str(), device_id, initial_device_display_name).await
     }
 
+    /// Login as appservice virtual user
```

Comment:
```suggestion
    /// Login as an appservice virtual user
```

Reply:
What does this really mean? `appservice` is not an adjective. Should it be `appservice-generated`?